app.rb

```diff
@@ -1,7 +1,57 @@
 require 'sinatra/base'
+require_relative './lib/computer_choice'
+require_relative './lib/game'
+require_relative './lib/player'
+
```

Comment:
I think I've figured out what's going on but I was slightly confused by the lack of post routes for weapon choice.